src/Gaufrette/Adapter/Ftp.php

```diff
@@ -366,12 +378,31 @@ private function isDir($directory)
             return true;
         }
 
-        if (!@ftp_chdir($this->getConnection(), $directory)) {
-            return false;
+        $chDirResult = false;
+        try {
+            $chDirResult = ftp_chdir($this->getConnection(), $this->directory);
+
+            // change directory again to return in the base directory
+            ftp_chdir($this->getConnection(), $this->directory);
+        } catch(\Exception $e) {
```

Comment:
same here : can you add a space between the `catch` statement and the opening parenthesis please ?